blocks/components/editable/style.scss

```diff
@@ -13,13 +13,31 @@
 		outline: none;
 	}
 
+	a {
+		color: $blue-medium;
+	}
+
 	a[data-mce-selected] {
 		padding: 0 2px;
 		margin: 0 -2px;
-		background: #dcf2f8;
 		border-radius: 2px;
-		box-shadow: 0 0 0 1px #dcf2f8;
+		box-shadow: 0 0 0 1px $blue-medium-100;
+		background: $blue-medium-100;
+	}
+
+	code {
+		padding: 0 2px;
+		margin: 0 -2px;
+		border-radius: 2px;
+		color: $dark-gray-800;
+		background: $light-gray-200;
+		font-family: $editor-html-font;
+		font-size: 14px;
```

Comment:
Should we be so explicit about font size for code fragments? Should this be specific to only certain blocks instead? Currently it applies to e.g. Heading blocks, making for some odd visual appearance:

![image](https://user-images.githubusercontent.com/1779930/43787079-2b69d75e-9a38-11e8-884d-5b3cd6aa52ad.png)

Reply:
Good thought. Fixed in https://github.com/WordPress/gutenberg/pull/8914.